gateway-ha/pom.xml

```diff
@@ -221,6 +227,12 @@
             <artifactId>metrics-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-parser</artifactId>
+            <version>440</version>
```

Comment:
Use 445.

Reply:
445 causes `RequireUpperBoundDeps` errors when building, which I think is because gateway is using an earlier version of airlift. @oneonestar would upgrading airlift interfere with the work you are doing to remove Dropwizard?

Reply:
Unfortunately upgrading Airlift can't be done here. It looks like it will require removing dropwizard. 

Moving to airbase 156 required upgrading `jakarta.activation-api` to 3.0.0. This caused errors in CI like the below. I wasn't able to work around these, and some googling indicates that Dropwizard is simply incompatible with modern versions of jakarta. `SingleThreadModel` was [deprecated in 2.4](https://docs.oracle.com/cd/E17802_01/products/products/servlet/2.5/docs/servlet-2_5-mr2/javax/servlet/SingleThreadModel.html).
```
2024-05-22T04:24:30.916Z	INFO	main	stderr	java.lang.NoClassDefFoundError: jakarta/servlet/SingleThreadModel
2024-05-22T04:24:30.916Z	INFO	main	stderr		at org.eclipse.jetty.servlet.ServletHolder.setServlet(ServletHolder.java:173)
2024-05-22T04:24:30.917Z	INFO	main	stderr		at org.eclipse.jetty.servlet.ServletHolder.<init>(ServletHolder.java:146)
2024-05-22T04:24:30.917Z	INFO	main	stderr		at io.dropwizard.jetty.setup.ServletEnvironment.addServlet(ServletEnvironment.java:46)
2024-05-22T04:24:30.917Z	INFO	main	stderr		at io.dropwizard.core.setup.AdminEnvironment.<init>(AdminEnvironment.java:45)
2024-05-22T04:24:30.917Z	INFO	main	stderr		at io.dropwizard.core.setup.Environment.<init>(Environment.java:85)
2024-05-22T04:24:30.917Z	INFO	main	stderr		at io.dropwizard.core.cli.EnvironmentCommand.run(EnvironmentCommand.java:52)
2024-05-22T04:24:30.917Z	INFO	main	stderr		at io.dropwizard.core.cli.ConfiguredCommand.run(ConfiguredCommand.java:98)
2024-05-22T04:24:30.918Z	INFO	main	stderr		at io.dropwizard.core.cli.Cli.run(Cli.java:78)
2024-05-22T04:24:30.918Z	INFO	main	stderr		at io.dropwizard.core.Application.run(Application.java:94)
2024-05-22T04:24:30.918Z	INFO	main	stderr		at io.trino.gateway.ha.HaGatewayLauncher.main(HaGatewayLauncher.java:49)
2024-05-22T04:24:30.918Z	INFO	main	stderr		at io.trino.gateway.ha.security.TestAuthorization.setup(TestAuthorization.java:55)
```